src/cps-config/main.cpp

```diff
@@ -74,7 +81,13 @@ namespace cps_config {
         // clang-format on
         options.parse_positional({"package"});
         options.positional_help("<packages>");
-        auto parsed_options = options.parse(argc, argv);
+        cxxopts::ParseResult parsed_options;
+        try {
+            parsed_options = options.parse(argc, argv);
+        } catch (cxxopts::exceptions::no_such_option & exception) {
+            auto const debug_output = fmt::format("Error -- {}.\n\n{}\n", exception.what(), options.help({}, true));
```

Comment:
In cxxopts 3.0 this exception is called `cxxopts::option_not_exists_exception`, and options.help() doesn't take a second argument... so, I could only get this to work portably by:
```cpp
#if CXXOPTS__VERSION_MINOR == 0
        } catch (cxxopts::option_not_exists_exception & exception) {
            auto const debug_output = fmt::format("Error -- {}.\n\n{}\n", exception.what(), options.help({}));
#else
        } catch (cxxopts::exceptions::no_such_option & exception) {
            auto const debug_output = fmt::format("Error -- {}.\n\n{}\n", exception.what(), options.help({}, true));
#endif
```

Reply:
Yeah, I just found this myself because I didn't see your post yet.

I've been thinking we'll want subcommands and such anyway. Maybe we should move to a library that doesn't make API  changes like this a minor version bump.

I was thinking the simpler approach is to change the wraps to match the version in Ubuntu 22.04. I'll just do that for all the wraps while I'm at it.